govc/vm/keystrokes.go

```diff
@@ -156,7 +521,7 @@ func (cmd *keystrokes) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.VirtualMachineFlag.Register(ctx, f)
 
 	f.StringVar(&cmd.UsbHidString, "s", "", "Raw String to Send")
-	f.StringVar(&cmd.UsbHidCode, "c", "", "USB HID Code (hex)")
+	f.StringVar(&cmd.UsbHidCodes, "c", "", "USB HID Codes (hex), comma separated")
```

Comment:
Can we change this to "...or hex alias" or similar?

Reply:
Thanks. And thank you for the work you did on govc in the first place!